internal/cmd/flink/command_iam_binding_create.go

```diff
@@ -0,0 +1,79 @@
+package flink
+
+import (
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/errors"
+	"github.com/confluentinc/cli/internal/pkg/examples"
+	"github.com/confluentinc/cli/internal/pkg/output"
+)
+
+func (c *command) newIamBindingCreateCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create",
+		Short: "Create a Flink IAM binding.",
+		Args:  cobra.NoArgs,
+		RunE:  c.iamBindingCreate,
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: `Create a Flink IAM binding for AWS region "us-west-2" and environment "env-123".`,
```

Comment:
Not sure what the proper terminology is here, but maybe we should say `Create an IAM binding for identity pool "pool-123" in AWS region "us-west-2"` instead? I thought the binding was in terms of the identity pool, not the environment.

Reply:
It's a binding between env, region and an identity-pool. Changed the description to include the identity pool.